lib/queue-msg/src/lib.rs

```diff
@@ -408,16 +376,16 @@ impl<T: QueueMessage> Op<T> {
                         receiver.handle(store, data).await.map(Some)
                     }
                 }
-                Op::Repeat { times: None, msg } => {
-                    Ok(Some(seq([*msg.clone(), repeat(None, *msg)])))
-                }
-                Op::Repeat {
-                    times: Some(times),
-                    msg,
-                } => Ok(Some(seq([*msg.clone()].into_iter().chain(
-                    // if times - 1 > 0, queue repeat with times - 1
-                    NonZeroU64::new(times.get() - 1_u64).map(|times| repeat(Some(times), *msg)),
-                )))),
+                // Op::Repeat { times: None, msg } => {
+                //     Ok(Some(seq([*msg.clone(), repeat(None, *msg)])))
+                // }
+                // Op::Repeat {
+                //     times: Some(times),
+                //     msg,
+                // } => Ok(Some(seq([*msg.clone()].into_iter().chain(
+                //     // if times - 1 > 0, queue repeat with times - 1
+                //     NonZeroU64::new(times.get() - 1_u64).map(|times| repeat(Some(times), *msg)),
+                // )))),
```

Comment:
we ditch `repeat` and `race` completely?

Reply:
yes, forgot to remove this part fully - but imo these can both be implemented more efficiently for whatever specific usecase the functionality is needed for. removing these also removes significant complexity from the `Op` type and reduces potential nesting.